README.md

```diff
@@ -77,51 +79,55 @@ By contributing to this repository you agree to be a basic human being, please s
 
 ***TL;DR***: Initially during the early phase of development we'll keep this simple, after the library starts to become a dependency for real projects this will likely change.
 
-* Express interest and get added to the libdogecoin team on GitHub 
+- Express interest and get added to the libdogecoin team on GitHub
   and join the conversation in the Foundation discord server.
-* Branch/PRs in this repository (see above point for access)
-* Rebasing not merging
-* Ensure tests
-* Document how Dogecoin works as each feature is developed in [`/doc/spec`](doc/spec)
-* 1 approval from another contributor required to merge to main
-* Don't introduce dependencies without discussion (MIT)
-* Collaborate before you innovate! 
-* Have fun <3
+- Branch/PRs in this repository (see above point for access)
+- Rebasing not merging
+- Ensure tests
+- Document how Dogecoin works as each feature is developed in [`/doc/spec`](doc/spec)
+- 1 approval from another contributor required to merge to main
+- Don't introduce dependencies without discussion (MIT)
+- Collaborate before you innovate!
```

Comment:
+: "this means, discuss what you're working on where everyone can see, before submitting a change."